addons/data-category-tweaks-v2/userscript.js

```diff
@@ -184,19 +186,11 @@ export default async function ({ addon, console, msg, safeMsg }) {
     return result;
   };
 
-  // If editingTarget is set, the editor has already rendered and we have to tell it to rerender.
-  if (vm.editingTarget) {
-    vm.emitWorkspaceUpdate();
-  }
-
   addon.settings.addEventListener("change", (e) => {
-    // When the separate list category option changes, we need to do a workspace update.
+    // When the separate list category option changes, we need to update the toolbox XML.
     // For all other options, just refresh the toolbox.
-    // Always doing both of these in response to a settings change causes many issues.
```

Comment:
I removed this comment because I don't know what the "many issues" could be other than the ones this PR fixes.